presentation/charge_transaction.go

```diff
@@ -97,6 +107,24 @@ func (p *Presenter) ChargeTransaction(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
+		// if kind of error is RequestValidationError
+		if errors.As(err, &business.RequestValidationError{}) {
```

Comment:
ga gini cara pake errors.As. isi errornya sebaiknya diliat. jangan di assign ke empty struct.

```suggestion
      var requestValidationError *business.RequestValidationError
		if errors.As(err, &requestValidationError) {
```

abis itu dipake value requestValidationError nya

Reply:
beda ya antara
` var requestValidationError *business.RequestValidationError`
sama
`&business.RequestValidationError`

Reply:
beda. kalo yang pointer ke empty struct itu ya artinya nggak di pake value dari error nya